Packs/SplunkPy/Integrations/SplunkPy/SplunkPy_description.md

```diff
@@ -9,21 +9,21 @@ this will set the integration to use basic authentication when connecting to the
 For example:
 TestUser@_basic
 
-There are two main use cases for the SplunkPy integration. Refer to the following sections based on your needs.
-- [Splunk Enterprise Security Users](#splunk-enterprise-security-users)
-  - [Fetching notable events](#fetching-notable-events)
-    - [How to configure](#how-to-configure)
-    - [Enriching Notable Events](#enriching-notable-events)
-      - [Enrichment types](#enrichment-types)
-      - [How to configure](#how-to-configure-1)
-      - [Troubleshooting enrichment status](#troubleshooting-enrichment-status)
-      - [Resetting the enriching fetch mechanism](#resetting-the-enriching-fetch-mechanism)
-      - [Limitations](#limitations)
-    - [Incident Mirroring](#incident-mirroring)
-    - [Existing users](#existing-users)
-- [Splunk non-Enterprise Security Users](#splunk-non-enterprise-security-users)
-    - [Configure Splunk to Produce Alerts for SplunkPy for non-ES Splunk Users](#configure-splunk-to-produce-alerts-for-splunkpy-for-non-es-splunk-users)
-    - [Constraints](#constraints)
+There are two main use cases for the SplunkPy integration. 
+- Splunk Enterprise Security Users
+  - Fetching notable events
+    - How to configure
+    - Enriching Notable Events
+      - Enrichment types
+      - How to configure
+      - Troubleshooting enrichment status
+      - Resetting the enriching fetch mechanism
+      - Limitations
+    - Incident Mirroring
+    - Existing users
+- Splunk non-Enterprise Security Users
+    - Configure Splunk to Produce Alerts for SplunkPy for non-ES Splunk Users
+    - Constraints
```

Comment:
Let's replace all the anchor links (like #enrichment-types) with the absolute link
for example: #enrichment-types -> https://xsoar.pan.dev/docs/reference/integrations/splunk-py#enrichment-types
like we do already for a lot of integrations 

<img width="260" alt="image" src="https://github.com/demisto/content/assets/79846863/a09ccd41-9b50-4f5e-9e36-5e732301a0bf">